clients/vault/Cargo.toml

```diff
@@ -23,60 +23,59 @@ allow-debugger = ["console-subscriber"]
 
 
 [dependencies]
-async-trait = "0.1.40"
-base64 = { version = '0.13.0', default-features = false, features = ['alloc'] }
-bincode = "1.3.3"
-clap = { version = "3.1", features = ["env"] }
-cfg-if = "1.0.0"
-rand = { version = "0.8.5", optional = true }
-futures = "0.3.5"
-governor = "0.5.0"
-hex = "0.4.2"
-itertools = "0.10.5"
-lazy_static = "1.4"
-nonzero_ext = "0.3.0"
-reqwest = { version = "0.12.4", features = ["json"] }
-signal-hook = "0.3.14"
-signal-hook-tokio = { version = "0.3.1", features = ["futures-v0_3"] }
-sysinfo = "0.26.1"
-thiserror = "1.0"
+async-trait = { workspace = true}
+base64 = { workspace = true, features = ['alloc'] }
+bincode = { workspace = true }
+clap = { workspace = true, features = ["env"] }
+cfg-if = { workspace = true }
+rand = { workspace = true, optional = true }
+futures = { workspace = true }
+governor = { workspace = true }
+hex = { workspace = true, default-features = true }
+itertools = { workspace = true }
+lazy_static = { workspace = true }
+nonzero_ext = { workspace = true }
+reqwest = { workspace = true, features = ["json"] }
+signal-hook = { workspace = true }
+signal-hook-tokio = { workspace = true, features = ["futures-v0_3"] }
+sysinfo = { workspace = true }
+thiserror =  { workspace = true }
 
 # tokio related
-tokio = { version = "1.37", features = ["full", "tracing"] }
-tokio-metrics = { version = "0.1.0", default-features = false }
-tokio-stream = { version = "0.1.9", features = ["sync"] }
-console-subscriber = { version = "0.2.0", optional = true }
+tokio = { workspace = true, features = ["full", "tracing"] }
+tokio-metrics = { workspace = true }
+tokio-stream = { workspace = true, features = ["sync"] }
+console-subscriber = { workspace = true, optional = true }
 
-tracing = { version = "0.1", features = ["log"] }
+tracing = { workspace = true, features = ["log"] }
 
-jsonrpc-core-client = { version = "18.0.0", features = ["http", "tls"] }
+jsonrpc-core-client = { workspace = true, features = ["http", "tls"] }
 
 subxt = { version = "0.33.0", features = [ "substrate-compat"] }
 
+# Substrate dependencies
+# was with default-features = true??
```

Comment:
This shouldn't matter as we are in the client here. The client does not need to avoid `default-features` or `std` for that matter.

Reply:
Good find, it's actually wrong now.